lib/archethic/mining/pending_transaction_validation.ex

```diff
@@ -512,22 +512,31 @@ defmodule Archethic.Mining.PendingTransactionValidation do
        do: {:error, "No recipient specified in code approval"}
 
   defp do_accept_transaction(
-         %Transaction{
-           type: :keychain,
-           data: %TransactionData{content: "", ownerships: []}
-         },
+         %Transaction{type: :keychain, data: %TransactionData{content: ""}},
```

Comment:
You can factorize the two errors function into one.

Reply:
Or even better, you can use guard in the keychain acceptance to allow only tx with ownerships and contents, not transfers, etc..., the remaining can be discarded.